website/views.py

```diff
@@ -79,15 +80,13 @@ def index():
     try:
         #TODO : make this way more robust
         inst = Institution.find_one(Q('domain', 'eq', request.host))
```

Comment:
Do we store `domain` lower-cased? If so, you'll need to do `request.host.lower()`